libparsec/crates/client_connection/src/authenticated_cmds/mod.rs

```diff
@@ -228,7 +228,7 @@ impl AuthenticatedCmds {
         log::trace!("Will listen for SSE event at {}", response.url());
         let mut sse_stream = response.bytes_stream().eventsource();
         if let Some(last_event_id) = last_event_id {
-            sse_stream.set_last_event_id(last_event_id);
+            sse_stream.set_last_event_id(last_event_id.to_str().expect("last-event-id first originated from a String so converting it back should not cause problem"));
```

Comment:
make a method instead, so we can avoid explaining next time

Reply:
done

Reply:
I do not see this change applied, did I missed something? :thinking:

Reply:
Oh it was moved, sorry :sweat_smile: